src/devices/Common/ClassVisibility.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+// These instructions make the classes public for building the individual projects.
+// The default visibility is "internal" so they stay hidden if this section is removed for the final library build.
+#if !BUILDING_IOT_DEVICE_BINDINGS
+
+#pragma warning disable SA1403 // File may only contain a single namespace
+
+public partial class Interop
+{
+}
```

Comment:
consider putting these in a single line:

```suggestion
public partial class Interop { }
```

Reply:
This is not allowed by the current StyleCop rules. I could disable it here, but I'd rather keep it like this.